cmd/virt-v2v/entrypoint.go

```diff
@@ -2,279 +2,76 @@ package main
 
 import (
 	_ "embed"
-	"encoding/json"
-	"errors"
 	"fmt"
-	"io"
 	"os"
-	"os/exec"
-	"strconv"
-	"strings"
 
-	"github.com/konveyor/forklift-controller/pkg/virt-v2v/customize"
-	"github.com/konveyor/forklift-controller/pkg/virt-v2v/global"
+	"github.com/konveyor/forklift-controller/pkg/virt-v2v/config"
+	"github.com/konveyor/forklift-controller/pkg/virt-v2v/conversion"
 	"github.com/konveyor/forklift-controller/pkg/virt-v2v/server"
 	utils "github.com/konveyor/forklift-controller/pkg/virt-v2v/utils"
 )
 
 func main() {
-	var err error
-	if err = virtV2VPrepEnvironment(); err != nil {
-		fmt.Println("Failed to prepare the environment", err)
+	env := &config.AppConfig{}
+	err := env.Load()
+	if err != nil {
+		fmt.Println("Failed to load variables", err)
 		os.Exit(1)
 	}
-
-	// virt-v2v or virt-v2v-in-place
-	if _, found := os.LookupEnv("V2V_inPlace"); found {
-		err = runVirtV2vInPlace()
-	} else {
-		err = runVirtV2v()
+	if err = virtV2VPrepEnvironment(env); err != nil {
```

Comment:
virtV2VPrepEnvironment is a generic name, maybe two methods, one for ca certs and another for dir setup ?